packages/waci-pex/README.md

```diff
@@ -0,0 +1,45 @@
+<!--suppress HtmlDeprecatedAttribute -->
```

Comment:
I am missing code examples showing how this plugin can / should be used

Reply:
I do not have the example code good enough to be added here.

Reply:
Not sure how i should read your comment. 

Looking at the readme i have no clue how to use this. Your comment indicates you have example code, just make it decent? Also there should be code available when you did your own tests that is atleast a good start

Reply:
This plugin is even based of a package i wrote which should have a decent readme you can steal from ;)

Reply:
readme updated with example